### ISSUE TITLE
Achievements in Flixel

### ISSUE THREAD

**Issue by Tal3D** [FlixelCommunity/flixel#203]
Something thats often needed in games and helps highten the replay value: **Achievements**.

Though it is that important for flash games there is no implementation for it in flixel yet. I had to create it myself. [Check out my take on it.](http://github.com/tal3d/flxachievementsystem)
Even when you dont want to officially implement it in this flixel release it will probably be something that could be helpful to other flixel developers.

**Comment by Dovyski** [comment]
Your plugin looks really cool, thanks for the contribution! I think it fits perfectly as a plugin, since not all developers will need it.

As we evolve the project, eventually a plugin repository will come to life, something like a centralized place to find Flixel code. Right now we don't have any 3rd party plugin shipped with Flixel.

**Comment by greysondn** [comment]
Dovyski, is it impossible to link these as they come in via the WWW / git webpage?

I realize that's an ad-hoc solution for something that may come later, "but..." it's perhaps worth encouraging all reasonable contributions for the short term and linking users to developers better - even third party developers - to help grow the same ecosystem you're talking about.

Off-topic: I'm way behind and will be catching up in the coming days. I apologize as these notifications reach you if I find something worth noting or saying in these issues in the tracker.

**Comment by Dovyski** [comment]
Yeah, I totally agree. As soon as I have some free time, I will create a temporary plugins area in the site. The task is related to [#5](https://github.com/FlixelCommunity/flixelcommunity.github.com/issues/5).

> Off-topic: I'm way behind and will be catching up in the coming days. I apologize as these notifications reach you if I find something worth noting or saying in these issues in the tracker.

Ok, no problem!
